### PR TITLE
Add docker-compose.yaml

### PR DESCRIPTION
**Pull request by jackzampolin**
Add a docker compose file and instructions on how/why to run. Also move `blockchain` into `data/blockchain`.

cc @alexanderbez and @AlexeyAkhunov

### CODE REVIEW

**Comment by alexanderbez** [comment]
@jackzampolin tbh, I don't think we need this atm. This is really just meant for @AlexeyAkhunov and I to quickly smoke test. Eventually the docker-compose file with actually start a local few node ethermint network.

**Comment by jackzampolin** [comment]
@alexanderbez I get that. I would also like to be able to quickly smoke test, and having those instructions on the repo would be great. They don't need to be through a `docker-compose` however so I can pull that out, but I'm happy to continue updating that as work progresses.

**Comment by alexanderbez** [comment]
@jackzampolin yeah that's cool with him. Remove the compose file (for now) and just keep the instructions in the README 👍

**Comment by jackzampolin** [comment]
Fixed!

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@
 .glide/
 vendor/
 build/
+*.db
```
Comment (alexanderbez) [inline .gitignore]: I think we can avoid both of these. We need to allow this to be configurable (the datadir)

File: README.md [hunk 2]
```diff
@@ -33,10 +33,28 @@ $ make tools deps install
...
 ### Using Ethermint to parse Mainnet Ethereum blocks
 
-There is an included `docker-compose.yaml` file and a `data/blockchain` file that provide an easy way to run the demo of parsing Mainnet ethereum blocks. The dump in `data/` only includes up to block `97638`. To run this have [`docker-compose.yaml` installed](https://docs.docker.com/compose/install/) and run the following:
+There is an included Ethereum Mainnet blockchain file in `data/blockchain` that provides an easy way to run the demo of parsing Mainnet Ethereum blocks. The dump in `data/` only includes up to block `97638`. To run this, type the following command:
 
-```
-$ docker-compose up
+```bash
+$ go run main.go copied.go
+balance of 0x756F45E3FA69347A9A973A725E3C98bC4db0b5a0: 200000000000000000000
```
Comment (alexanderbez) [inline README.md]: I think having the command is enough for now. This was all thrown together rather quickly, so I wouldn't rely or worry much about the exact output.

**Review by alexanderbez — CHANGES_REQUESTED** [review]
Thanks @jackzampolin, left two minor suggestions 👍

File: README.md (hunk 2 above)
Comment (jackzampolin) [reply README.md]: Done!

File: .gitignore (hunk 1 above)
Comment (jackzampolin) [reply .gitignore]: When I ran the command I got all that stuff showing up in my project dir. I was just adding them so they don't get picked up in git history. Is that an issue?

Comment (alexanderbez) [reply .gitignore]: Yeah cause that's where Alexey has them by default. I don't want to pollute this file with that stuff unless it's going to be permanent (which its not). You have two options:

1. Just keep deleting them
2. Add them to your personal global gitignore:
```
$ git config --global core.excludesfile ~/.gitignore_global
```
`~/.gitignore_global` will contain all the stuff you dont wanna commit 👍

Comment (jackzampolin) [reply .gitignore]: pulled them out and added issue to track data directory work: https://github.com/cosmos/ethermint/issues/430

**Review by alexanderbez — APPROVED** [review]
Thans @jackzampolin 🙌

**Pull request merged**